burn-book/src/building-blocks/tensor.md

```diff
@@ -20,10 +20,12 @@ The actual shape of the tensor is inferred from its initialization. For example,
 ```rust, ignore
 let floats = [1.0, 2.0, 3.0, 4.0, 5.0];
 
+let device = burn::backend::wgpu::WgpuDevice::default();
```

Comment:
Instead of making the device specific to wgpu, we can make the example generic to the backend.

Use this instead:
```rust
let device = Default::default();
```

It will automatically select the default device on the selected backend.

Reply:
Wouldn't Default have to be imported using a use something? 

When I try it on example code I get an error 

```
no function or associated item named `device` found for trait `std::default::Default`
function or associated item not found in `Default`
```

Reply:
Copying the answer from our discord convo:

Not required, no. The compiler is able to figure out which default trait implementation to call based on the rest of the code (type inference). `Default::default()` is just `<_ as Default>::default()`

Try it for yourself:
```rust
use burn::{backend::Wgpu, tensor::Tensor};

// Type alias for the backend to use.
type Backend = Wgpu;

fn main() {
    // correct: Tensor is 1-Dimensional with 5 elements
    let device = Default::default();
    let tensor_1 = Tensor::<Backend, 1>::from_floats([1.0, 2.0, 3.0, 4.0, 5.0], &device);

    println!("{}", tensor_1);
}
```

Not sure in what context/program you're trying to use this snippet, but the example usage in this markdown are not meant to be complete with all modules declaration anyway. If you're using it like the snippet I wrote above it should work :)

Reply:
Thanks I understand. I have made the fixes and pushed